src/client/pages/Path/Path.jsx

```diff
@@ -21,18 +22,6 @@ const CourseCard = ({ course, linkTo }) => (
   </Link>
 );
 
-const ItemInfoCard = ({ item }) => (
```

Comment:
This component uses 6 custom classes, if we add the 2 more from `lesson` and `course` we reach 8 classes for a simple panel.

I replicated the same exact style using 3 modular classes and sharing the same component.